include/crm/fencing/internal.h

```diff
@@ -105,7 +105,10 @@ void stonith__device_parameter_flags(uint32_t *device_flags,
                                      const char *device_name,
                                      xmlNode *metadata);
 
-#  define ST_LEVEL_MAX 10
+/* Only 1-9 is allowed for fencing topology levels,
+ * however, 0-10 has been used internally.
```

Comment:
FYI, in unregister requests, 0 means unregister all levels. I believe that's the only place 0 is allowed.